README.md

```diff
@@ -61,6 +61,12 @@ The mod config is located in `cfg.json`.  You can change this config to your lik
 		* Other special monsters at approximately 200
 		* All other monsters at approximately 450
     * Set to `-1` to omit
+* **RuneDropRate** `float`
```

Comment:
need to also update the example cfg in the readme with the default value as well as the cfg.json with the default value